trigger/version.go

```diff
@@ -193,7 +193,23 @@ func (t *versionTriggers) versionGithubCheckOutcome(sub *event.Subscription) (*n
 }
 
 func (t *versionTriggers) versionFailure(sub *event.Subscription) (*notification.Notification, error) {
-	if t.data.Status != evergreen.VersionFailed {
+	versionTasks, err := task.FindAll(task.ByVersion(t.version.Id))
+	if err != nil {
+		return nil, errors.Wrap(err, "error getting tasks in version")
+	}
+	skipNotification := false
+	// If the only failures in the version come from aborted tasks then we skip sending a notification
+	for _, versionTask := range versionTasks {
+		if versionTask.Status == evergreen.TaskFailed {
+			if versionTask.Aborted {
+				skipNotification = true
+			} else {
+				skipNotification = false
+				break
+			}
+		}
+	}
+	if t.data.Status != evergreen.VersionFailed || skipNotification {
```

Comment:
We should do this before 196; if the version isn't failed, we already know we don't have an aborted task